test/corpus/statements.txt

```diff
@@ -1526,3 +1526,202 @@ end
               method: (identifier)
               arguments: (argument_list))
             method: (identifier)))))))
+
+==================================
+case stmt with pseudo impl methods
+:error
```

Comment:
This should succeed once #76 is merged, right?

Reply:
Can you group with the rest of the `case` tests in `test/corpus/expressions.txt`?

Reply:
IIRC no, additional work needs to be done to support implicit pseudo methods

Reply:
Ohhh I didn't catch that -- should stay as a TODO then, right? I have a branch from a couple weeks ago that improves how implicit objects work, maybe it's time to bring that back.